.fixtures.yml

```diff
@@ -1,6 +1,8 @@
 fixtures:
   repositories:
-    apt: "https://github.com/puppetlabs/puppetlabs-apt.git"
+    apt:
+      repo: "https://github.com/puppetlabs/puppetlabs-apt.git"
+      ref: "v7.7.0"
```

Comment:
Shouldn't the commit be in git as well?

Reply:
It is, i just added the latest released version to see how it performs.

Reply:
But how are you going to determine a baseline for this repository? Since it was merged in April, all CI should have run with it for months already. Are you going to open a PR pinning it to puppetlabs-apt v7.4.2?

Reply:
I was looking at the wrong logs, this was already running with the latest module ref. So this change is pointless 😅